raystack/optimus
# feat: optimus upgrades

-  fix: user name_template for goreleaser 1.19
- chore: update version of goreleaser for publish latest
- feat: differentiate pools for task, hook, sensors
- feat: sync job api
- feat : add job disable API
- fix: scheduler event logging for better monitoring debugging
- feat: refresh on resource update
- feat: immediate sensor to fail on non success when fetching the job run status
- feat: resource apply to sync resources with datastore
- fix : sla callback notification appropriate schedule time
- feat: add authentication with oauth
- feat: add support for using https to connect to optimus server
- fix: airflow client api
- fix: concurrent read write issue in metric map
- refactor: simplify optimus plugin
- (tag: v0.8.2) Revert "feat: simplify plugin proto
- (tag: v0.8.1) fix: SLA to be at DAG level
- feat: job spec name validation
- refactor: standardize metrics and add documentation
- feat: improve user experience replay
- feat: continually poll replay status
- feat: simplify plugin proto
- remove: catchup capability
- (tag: v0.8.0) Add monitoring role and update dashboard json model 
- Feat : resource namespace change
- (tag: v0.7.3-rc.1) Feat: job namespace change 
- feat: startup timeout for operator creation
- feat: publish missing metrics
- fix: cast grpc numbers to float64
- fix: upsert job run
- fix: return all validation errors on check job spec
- refactor: log placement
- feat: capability to create runs on replay
- fix: unable to delete upstreams if no longer upstream found issue
- fix: replay status runs ordering
- Fix: optimus job failure alerts 
- fix: show wrapped error when calling method error
- fix: test unique label values using element match
- remove: catchup capability
- fix: XCom state logic
- refactor: remove unused libs
- refactor: remove old priority resolver
- fix: provide upload jobs with simple priority resolver
- fix: window with playground
- fix: duplicate tags label
- (tag: v0.7.1) fix: use simple priority resolver
- refactor: trace the upload to scheduler
- (tag: v0.7.0) fix: update job state query main
- fix: update kafka write for errors
- refactor: use no-op handler for no publisher
- feat: Publish job run events
- feat: job downstream deletion
- fix: add migration to cleanup old , data and remove delete cascade fr…
- refactor: error handling changes
- feat: improve check job specs
- (tag: v0.7.0-rc.1) fix: do not return error in case resource already exists
- fix: upload to scheduler async
- fix: disable depends_on_past for sensors and hooks
- fix: job scheduling configs not match
- refactor: propogate template errors as invalid argument
- refactor: reduce concurrency
- feat: publish job events
- Publish events extended
- fix: replay issues
- fix: debug log
- docs: update documentation to the latest 0.6 version
- feat: get replay status
- feat: get replay list
- fix: allow resources to be uploaded in batches
- fix: revert get window handler default version to 1
- chore: fix imports and proto commit
- feat: add replay
- fix: fix init container url
- fix: override depends_on_past's dag level value
- fix: remove snapshot mode in publish latest
- fix: fix build by using new version of salt
- refactor: update protos and remove notice
